### PR TITLE
feat(article): Peaking at A/B testing

### PR DESCRIPTION
**Pull request by sanderdejong88**

### CODE REVIEW

File: data/authors/abdelhak-chahid.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+name: Abdelhak Chahid
+avatar: /authors/abdelhak-chahid.jpeg
+occupation: Data Scientist
+medium: https://medium.com/@Abdelhak_chahid
+linkedin: https://www.linkedin.com/in/abdelhak-chahid-2929aa1b7/
```
Comment (DaveBitter) [inline data/authors/abdelhak-chahid.md:6]: This renders a bit weird in the sidebar. Can we do some replace magic to make it look nice?

File: data/blog/peaking-at-ab-testing.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+In this article, we discussed why continuous monitoring violates the existing paradigm within the frequentist approach and leads to invalid inference. Furthermore, we have demonstrated empirically the pitfalls of continuous monitoring.
+
+The online platform on the internet has changed the paradigm of A/B testing and has led to the following challenge: is it possible to develop a method and deploy it into a platform where the user will be continuously monitoring experiments whereas the false positive and experiment runtime will still be kept under control?
+
+In the next articles, we will touch upon some suggested methods that allow users to control the false positive rate and improve the user’s ability to tradeoff between detection of real effects and the length of the experiment, while continuously monitoring the experiment.
+
+GitHub repository for the Python code: [https://github.com/achahid/A_B_testingPeekingProblem](https://github.com/achahid/A_B_testingPeekingProblem)
+
+## Some basic statistical concepts:
```
Comment (DaveBitter) [inline data/blog/peaking-at-ab-testing.md]: Can we link to the explanation in the article where they are used by using a hash link?

Comment (sanderdejong88) [reply data/blog/peaking-at-ab-testing.md]: footnodes!

File: data/authors/abdelhak-chahid.md (hunk 1 above)
Comment (sanderdejong88) [reply data/authors/abdelhak-chahid.md:6]: fixed

**Pull request merged**